planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module.hpp

```diff
@@ -300,33 +306,91 @@ class AvoidanceModule : public SceneModuleInterface
   void updateAvoidanceDebugData(std::vector<AvoidanceDebugMsg> & avoidance_debug_msg_array) const;
   mutable std::vector<AvoidanceDebugMsg> debug_avoidance_initializer_for_shift_line_;
   mutable rclcpp::Time debug_avoidance_initializer_for_shift_line_time_;
-  // =====================================
+
   // ========= helper functions ==========
-  // =====================================
+
+  double getEgoSpeed() const
```

Comment:
What is the reason for moving implementation to header?

Reply:
The number of lines in avoidance_module.cpp exceeds 3000. So I move simple helper fuction definition to header file in order to reduce the volume of cpp file.

Reply:
I got it. Or you can create another file avoidance_module_helper.cpp and put them there.

Reply:
Nice idea :+1: I'll do that in refactoring ticket.